client/app/resources/i18n/privateDatabase/Messages_fr_CA.xml

```diff
@@ -514,7 +514,7 @@
    <translation id="privateDatabase_modale_whitelist_add_title" qtlid="405019">Ajouter une adresse IP / masque</translation>
    <translation id="privateDatabase_modale_whitelist_add_info" qtlid="405058">Ajouter une nouvelle IP / masque à votre whitelist</translation>
    <translation id="privateDatabase_modale_whitelist_add_success" qtlid="405071">La création de la whitelist a été prise en compte</translation>
-   <translation id="privateDatabase_modale_whitelist_add_fail" qtlid="405084">Une erreur est survenue lors de la céation de la whitelist.</translation>
+   <translation id="privateDatabase_modale_whitelist_add_fail" qtlid="405084">Une erreur est survenue lors de la création de la whitelist.</translation>
```

Comment:
Due to internal process, you should update only the `Messages_fr_FR.xml` as specified in the contributing file (https://github.com/ovh-ux/ovh-manager-web/blob/master/.github/CONTRIBUTING.md)